src/app.js

```diff
@@ -3,14 +3,23 @@ import Router from './router'
 import styles from './css/main.styl'
 import bootstrap from "../node_modules/bootstrap/dist/css/bootstrap.css"
 import Me from './models/me'
+import Firebase from 'firebase'
 
 window.app = app
 
 app.extend({
   init () {
-    this.me = new Me()
-    this.router = new Router()
-    this.router.history.start()
+    this.firebase = new Firebase("https://brilliant-inferno-9726.firebaseio.com/")
+    this.firebase.child('repos').on('value', (snapshot) => {
+      let val = snapshot.val()
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Missing semicolon.